apps/home/templates/apps_home/overview_page.html

```diff
@@ -19,8 +19,8 @@ <h1>{{ page.page_title }}</h1>
     {{ self.page_intro|richtext }}
 </section>
 <div id="layout-grid__area--maincontent">
-    {% for teaser in self.teasers %}
```

Comment:
@phillimorland ah, probably this is why it errored, because overview page was still using it then!

Reply:
ahhh cool, ok will do it all in 1 and redo migrations and see